tools/builder/build_mfx.pl

```diff
@@ -257,32 +263,30 @@ sub _opt_cmake_handler {
 my $build_root = getcwd;
 my $target_path = "$build_root/__cmake/" . get_cmake_target(%build);
 
+my $cmake_gen_cmd;
 # "make" and "eclipse"" generators output directory is different
 if ($build{'generator'} =~ /^eclipse$/) {
-    my $target_path_eclipse_case_bin_folder = $build_root . "/__bin";
-    my $target_path_eclipse_case_lib_folder = $build_root . "/__lib";
-
-    if ($clean) {
-        rmtree($target_path_eclipse_case_bin_folder) if -d $target_path_eclipse_case_bin_folder;
-        rmtree($target_path_eclipse_case_lib_folder) if -d $target_path_eclipse_case_lib_folder;
-    }
+    my $mfx_home_path = nativepath($ENV{MFX_HOME});    
+    $target_path = "$build_root/" . get_cmake_target(%build);
+    rmtree($target_path) if -d $target_path  and $clean;
+    mkpath($target_path) if !-d $target_path and -d $build_root;
+    #chdir($target_path) or die "can't chdir -> $target_path: $!";
+    chdir($mfx_home_path) or die "can't chdir -> $target_path: $!";
+    $cmake_gen_cmd = get_cmake_gen_cmd($target_path, %build);
 } else {
     rmtree($target_path) if -d $target_path  and $clean;
     mkpath($target_path) if !-d $target_path and -d $build_root;
     chdir($target_path) or die "can't chdir -> $target_path: $!";
+    $cmake_gen_cmd = get_cmake_gen_cmd($build_root, %build);
 }
 
 my $exit_code = 0;
-my $cmake_gen_cmd = get_cmake_gen_cmd($build_root, %build);
+#my $cmake_gen_cmd = get_cmake_gen_cmd($build_root, %build);
```

Comment:
Never understand why need comment code and leave it without explanation. Wouldn't it better to remove it?

Reply:
Fine. Let's remove